roles/openshift_health_checker/library/etcdlogs.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=missing-docstring
+
+"""Interface to journalctl"""
```

Comment:
there's actually nothing about this module that's specific to the task at hand; it could probably be renamed to search_journal.py or some such. although in a more generic implementation you'd probably search for several things at the same time in a single unit's journal. but we can cross that bridge later.